### PR TITLE
TRE-113 Fixed versioning and changed package name

### PR DESCRIPTION
**Pull request by VishnuSubramanianTrenser**

### CODE REVIEW

**Review by zvi-quantivly — CHANGES_REQUESTED** [review]
setup.py is deprecated. We should switch to pyproject.toml and update the metadata.

File: README.md [hunk 1]
```diff
@@ -58,18 +58,21 @@ getting the response with the return value.
...
 from fastapi import FastAPI
-from fastapi_websocket_rpc import RpcMethodsBase, WebsocketRPCEndpoint
+from fastapi_ws_rpc import RpcMethodsBase, WebsocketRPCEndpoint
+
 
 # Methods to expose to the clients
 class ConcatServer(RpcMethodsBase):
-    async def concat(self, a="", b=""):
-        return a + b
+  async def concat(self, a="", b=""):
```
Comment (zvi-quantivly) [inline README.md:70]: Is this right? It looks like indentation is wrong here, no?

Comment (VishnuSubramanianTrenser) [reply README.md:70]: 2 indent space is fine right?

Comment (zvi-quantivly) [reply README.md:70]: No. Python uses 4 spaces [[ref](https://peps.python.org/pep-0008/#indentation)].

Comment (VishnuSubramanianTrenser) [reply README.md:70]: Yeah correct, so I added 4, On the change request it seems 2, so I have changed the README.md to 2 indent. Can we follow 4 indent in README.md as well?

Comment (zvi-quantivly) [reply README.md:70]: Yes, everywhere should have 4.

File: pyproject.toml
Comment (zvi-quantivly) [inline pyproject.toml:1]: Let's conform with the rest of the Quantivly codebase and use Poetry.

**Review by zvi-quantivly — CHANGES_REQUESTED** [review]
Note that the four spaces fix is still not applied.

**Review by zvi-quantivly — APPROVED** [review]

**Review by zvi-quantivly — APPROVED** [review]

**Pull request merged**